scripts/build-package.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+import { createRequire } from 'module';
+import upath from 'upath';
+import { build } from '@ckeditor/ckeditor5-dev-build-tools';
+
+( async () => {
+	const require = createRequire( import.meta.url );
+	const pkg = require( upath.resolve( process.cwd(), './package.json' ) );
+
+	await build( {
+		input: 'src/index.ts',
+		output: upath.resolve( process.cwd(), './dist/index.js' ),
```

Comment:
TBH, I do not like that we depend on `process.cwd()`. I would see it as a CLI script - `bash build-package --path ...`.

Is it possible?

Reply:
This script is called from each package. This means that in each package we would have to specify the path to it, for example

```bash
# packages/ckeditor5-adapter-ckfinder
node ../../scripts/build-package.mjs --path=packages/ckeditor5-adapter-ckfinder

# packages/ckeditor5-alignment
node ../../scripts/build-package.mjs --path=packages/ckeditor5-alignment

# packages/ckeditor5-autoformat
node ../../scripts/build-package.mjs --path=packages/ckeditor5-autoformat

# etc...
```

This path could be `.join`-ed with one of the `constants'.

IMO this would create unnecessary noise.

Reply:
While I do not like depending on `process.cwd()`, I am convinced to leave it as it is.